freertos/pubnub_config.h

```diff
@@ -70,7 +70,7 @@
     in milliseconds. Timeout duration in the context can be changed by the 
     user after initialization.
     */
-#define PUBNUB_DEFAULT_TRANSACTION_TIMER    310000
+#define PUBNUB_DEFAULT_TRANSACTION_TIMER    1024
```

Comment:
Why did we reduce transactional request timeout (subscribe)?

Reply:
Leftover after testing. Gonna return to the previous value